README.md

```diff
@@ -70,7 +70,12 @@ PyRGG is a user-friendly synthetic random graph generator that is written in Pyt
 - Check [Conda Managing Package](https://conda.io)
 - `conda install -c sepandhaghighi pyrgg`
 
-### Exe Version (Only Windows)
+### Exe Version
+
+⚠️ Only Windows
+
+⚠️ For PyRGG targeting Windows < 10, the user needs to take special care to include the Visual C++ run-time .dlls: Python >=3.5 uses Visual Studio 2015 run-time, which has been renamed into “Universal CRT“ and has become part of Windows 10. For Windows Vista through Windows 8.1 there are Windows update packages, which may or may not be installed in the target-system
```

Comment:
`.dlls` instead of .dlls
Also this line is a very hard instruction. Can we reference somewhere else for that or make it easier?

Reply:
good point
fixed in https://github.com/sepandhaghighi/pyrgg/pull/180/commits/9ca00d91c1ebd4180b82a8a3e0764c18dfc3c2cc